rust-src/aggregate_sig/benches/bench.rs

```diff
@@ -89,6 +89,38 @@ fn bench_verify_aggregate_sig(c: &mut Criterion) {
     });
 }
 
+fn bench_verify_aggregate_sig_hybrid(c: &mut Criterion) {
```

Comment:
Please explain in a sentence or two the intent and scope of this benchmark. Why is it meaningful?

Reply:
Doc'd it.